### PR TITLE
COMMECE-1430 Minicart is aware of order workflow now

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by marco-leo** [comment]
@gianmarcobrunialti please fix the SF issue

**Comment by gianmarcobrunialti** [comment]
ci:test

File: commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java [hunk 1]
```diff
@@ -61,6 +62,10 @@ public int doStartTag() {
 				putValue("orderId", commerceOrder.getCommerceOrderId());
 				putValue(
 					"commerceAccountId", commerceOrder.getCommerceAccountId());
+				putValue(
+					"workflowStatus",
+					WorkflowConstants.getStatusLabel(
```
Comment (marco-leo) [inline commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java]: @gianmarcobrunialti  use the getStatus int not the label

**Review by marco-leo — CHANGES_REQUESTED** [review]

File: commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java (hunk 1 above)
Comment (gianmarcobrunialti) [reply commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/MiniCartTag.java]: Done 👍

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
Worth checking whether functional tests pass now.

**Review by ethanbustad — APPROVED** [review]

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

**Review by ethanbustad — CHANGES_REQUESTED** [review]
@gianmarcobrunialti can you just rename your commit? It says `COMMECE` instead of `COMMERCE` atm.

**Comment by gianmarcobrunialti** [comment]
@ethanbustad Just done 👍

**Review by ethanbustad — APPROVED** [review]

**Pull request merged**